package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rbxts/object-event",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Because this qualifies more as a bug-fix rather than a feature update, could you change this back to `1.0.1`?

Reply:
Yep, sounds right - thanks for the review!